### ISSUE TITLE
Fault creating process

### ISSUE THREAD

**Issue by brendandixon** [DarthSim/hivemind#12]
Just upgraded to the latest and running within a Docker container built from either the official Node 8.12 image or the official Ruby 2.6 image, after installing Go, Hivemind faults on startup:

```
panic: runtime error: invalid memory address or nil pointer dereference
react_1  | [signal SIGSEGV: segmentation violation code=0x1 addr=0x38 pc=0x402fee]
react_1  |
react_1  | goroutine 1 [running]:
react_1  | panic(0x565100, 0xc420010050)
react_1  | 	/usr/lib/go-1.7/src/runtime/panic.go:500 +0x1a1
react_1  | main.newProcess(0xc420010750, 0x3, 0xc420010755, 0xb, 0x2, 0xc4200105c0, 0x6, 0x1388, 0xc42000a800, 0xc4200105c0)
react_1  | 	/go/src/github.com/DarthSim/hivemind/process.go:32 +0x34e
react_1  | main.newHivemind(0x0, 0x0, 0x7ffedfba6edb, 0xc, 0x0, 0x0, 0xc4200105c0, 0x6, 0x1388, 0x64, ...)
react_1  | 	/go/src/github.com/DarthSim/hivemind/hivemind.go:52 +0x335
react_1  | main.main.func1(0xc42009c000, 0xc42009c000, 0xc420035a87)
react_1  | 	/go/src/github.com/DarthSim/hivemind/main.go:67 +0x16d
react_1  | github.com/DarthSim/hivemind/vendor/gopkg.in/urfave/cli%2ev1.HandleAction(0x55e720, 0xc42000a1a0, 0xc42009c000, 0x0, 0x0)
react_1  | 	/go/src/github.com/DarthSim/hivemind/vendor/gopkg.in/urfave/cli.v1/app.go:485 +0xd4
react_1  | github.com/DarthSim/hivemind/vendor/gopkg.in/urfave/cli%2ev1.(*App).Run(0xc4200981a0, 0xc42000a140, 0x2, 0x2, 0x0, 0x0)
react_1  | 	/go/src/github.com/DarthSim/hivemind/vendor/gopkg.in/urfave/cli.v1/app.go:259 +0x74f
react_1  | main.main()
react_1  | 	/go/src/github.com/DarthSim/hivemind/main.go:72 +0x6e9
```

All was good until the latest change.

**Comment by DarthSim** [comment]
My bad. Fixed in [v1.0.6](https://github.com/DarthSim/hivemind/releases/tag/v1.0.6)

**Comment by brendandixon** [comment]
Thanks.
